pyrasterframes/src/main/python/docs/getting-started.pymd

```diff
@@ -78,8 +78,41 @@ spark.stop()
 
 ## Installing GDAL 
 
-GDAL provides a wide variety of drivers to read data from many different raster formats. If GDAL is installed in the environment, RasterFrames will be able to @ref:[read](raster-read.md) those formats.  If you are using the @ref:[Jupyter Notebook image](getting-started.md#jupyter-notebook), GDAL is already installed for you. Otherwise follow the instructions below.
+GDAL provides a wide variety of drivers to read data from many different raster formats. If GDAL is installed in the environment, RasterFrames will be able to @ref:[read](raster-read.md) those formats.  If you are using the @ref:[Jupyter Notebook image](getting-started.md#jupyter-notebook), GDAL is already installed for you. Otherwise follow the instructions below. Version 2.4.1 or greater is required.
 
-__TODO__ Verify notebook env has GDAL? 
+### Installing on MacOS
 
-__TODO__ How to install it.
+Using [homebrew](https://brew.sh/):
```

Comment:
well this helped me a lot! Embarrasing! I have typically used conda to get gdal because it typically plays nice with python libraries, but this is great.